zerver/tests/tests.py

```diff
@@ -1548,6 +1548,24 @@ def check_for_toggle_param(self, pattern, param):
         user_profile = get_user_profile_by_email("hamlet@zulip.com")
         self.assertEqual(getattr(user_profile, param), False)
 
+    def check_for_toggle_param_patch(self, pattern, param):
```

Comment:
Don't there is an alternative. I think we might have to do this with left_side_userlist as well because it originally used check_for_toggle_param.